src/rebar_app_utils.erl

```diff
@@ -140,6 +140,8 @@ parse_deps(Parent, DepsDir, Deps, State, Locks, Level) ->
       Level :: non_neg_integer().
 parse_dep(Dep, Parent, DepsDir, State, Locks, Level) ->
     Name = case Dep of
+               [PkgName, _Vsn, _Required, _App] ->
+                   PkgName;
```

Comment:
We weren't using the hex registries entries before but creating our own.

Yea, probably a good idea to add that `|_]`